setup.sh

```diff
@@ -335,7 +335,7 @@ function _main
 
     config)
       shift ; case ${1:-} in
-        dkim     ) _docker_image generate-dkim-config "${2:-2048}" ;;
+        dkim     ) _docker_image generate-dkim-config "${2:-2048}" "${3}"
```

Comment:
I would like to see `${3:-}` here since the `set -u` from above will end the script immediately of `$3` is not provided, and this is only the case for LDAP setups. The tests should show this. This is rather intricate, and you could not possibly know this since this script has some intricate `set ...` options